libs/designer/src/lib/core/utils/parameters/dynamicdata.ts

```diff
@@ -663,6 +666,14 @@ function getSwaggerTypeFromVariableType(variableType: string): string | undefine
       return undefined;
   }
 }
+function getSwaggerEnumFromVariableType(variableType: string): boolean[] | undefined {
+  switch (variableType) {
+    case 'boolean':
+      return [true, false];
+    default:
+      return undefined;
+  }
+}
```

Comment:
I dont think this is needed for file/folder picker, can you remove this change

Reply:
Lets move it in separate PR, i need to understand that bug and review it separately.